annotations/utils.py

```diff
@@ -175,4 +175,20 @@ def _annotate_project_counts(queryset):
                                 Q(texts__relationsets__createdBy=F('ownedBy')),
                           distinct=True),
         num_collaborators=Count('collaborators', distinct=True)
-    )
\ No newline at end of file
+    )
+
+
+"""
+Helper function to sort collections alphabetically by name and integer parts (case-insensitive)
+"""
+def natural_keys(text):
```

Comment:
while I appreciate your thoughtfulness I don't think this is necessary, I doubt that the case we have in with test collections is very common. Let's keep this function since it might be coming in useful at some point, but let's sort collections just alphabetically.